content/contributing/tutorial-structure.md

```diff
@@ -0,0 +1,58 @@
+---
+html: tutorial-structure.html
+parent: contribute.html
+---
+
+# Tutorial Structure
+
+Each new XRPLF tutorial follows the same basic format.
+
+## Description
+
+1. A brief description of the features illustrated in the tutorial.
+
+![Description](img/tut-struct1.png)
```

Comment:
With the size of the images, they kind of break up the numbered list a bit much. I think it would read better if you moved the images to the start of each section (after the respective header) so they illustrate what that section is in more detail, which would then allow these bullet points to function as an overview of the scope/structure.

Reply:
+1 Agree that moving the images to the start of each section makes it easier to read.

Reply:
Good suggestion, moved the images.